packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1851,6 +1854,16 @@ class TimePickerDialog extends StatefulWidget {
   /// Callback called when the selected entry mode is changed.
   final EntryModeChangeCallback? onEntryModeChanged;
 
+  /// Optionnaly provide your own button style for the cancel button.
```

Comment:
typo; `Optionaly`

Reply:
In general though we avoid `you` and so this would be better written as:

```suggestion
  /// Button style for the cancel button.
```

Some more information would be good to add too, e.g. what happens when this changes (is it animated).

See also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc